cmake/linux/installer

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+[ "$1" ]
+
+VERSION="$1"
+DIR="$HOME"/.lmms-$VERSION
```

Comment:
I still don't get why we couldn't just offer downloads of .deb and .rpm packages. I'm just used to installing software with a .deb or by using a PPA. The home directory install sounds just confusing to me.